families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,378 @@
+              link:
+                $ref: "#/definitions/Link"
+        400:
+          $ref: "#/responses/400BadRequest"
+        404:
+          $ref: "#/responses/404NotFound"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+
+responses:
+  400BadRequest:
+    description: Request was malformed
+    schema:
+      $ref: "#/definitions/Error"
+  404NotFound:
+    description: Identifier did not match any resource
+    schema:
+      $ref: "#/definitions/Error"
+  500ServerError:
+    description: Something went wrong within the validator
+    schema:
+      $ref: "#/definitions/Error"
+  503ServiceUnavailable:
+    description: API is unable to reach the validator
+    schema:
+      $ref: "#/definitions/Error"
+
+parameters:
+  agent_id:
+    name: agent_id
+    in: path
+    type: string
+    required: true
+    description: Agent identifier, the public key the agent uses.
+  record_id:
+    name: record_id
+    in: path
+    type: string
+    required: true
+    description: "Record identifier, the serial number or natural identifier
+      of the item (material) being tracked."
+  count:
+    name: count
+    in: query
+    type: integer
+    default: 1000
+    description: Number of items to return
+  min:
+    name: min
+    in: query
+    type: string
+    description: Id or index to start paging (inclusive)
+  max:
+    name: max
+    in: query
+    type: string
+    description: Id or index to end paging (inclusive)
+
+definitions:
+  Link:
+    type: string
+    example: https://api.sawtooth.com/agents/65cd3a3ce088b265b626f704b7f3db97b6f12e848dccb35d7806f3d0324c71b709ed360d602b8b658b94695374717e3bdb4b76f77886953777d5d008558247dd
+  Paging:
+    properties:
+      start_index:
+        type: integer
+        example: 1000
+      total_count:
+        type: integer
+        example: 54321
+      previous:
+        type: string
+        example: https://api.sawtooth.com/agents?min=0&count=1000
+      next:
+        type: string
+        example: https://api.sawtooth.com/agents?min=2000&count=1000
+  Error:
+    properties:
+      code:
+        type: integer
+        example: 34
+      title:
+          type: string
+          example: Agent Not Found
+      message:
+        type: string
+        example: >
+          The Agent reqeusted was not found.
+  Agent:
+    properties:
+      identifier:
+        type: string
+        example: 02d260a46457a064733153e09840c322bee1dff34445d7d49e19e60abd18fd0758
+      name:
+        type: string
+        example: Bob Bobson
+  Application:
+    properties:
+      identifier:
+        type: string
+        example: 789-1232-9999-032
+      applicant:
+        type: string
+        example: 02d260a46457a064733153e09840c322bee1dff34445d7d49e19e60abd18fd0758
+      type:
+        type: string
+        example: OWNER
+        enum:
+          - OWNER
+          - CUSTODIAN
+      status:
```

Comment:
This appears to be the same value as the `state` parameter above. This value should only have one name. Status seems more meaningful to me.